content/reference/user/api/index.md

```diff
@@ -21,6 +21,7 @@ IDs can be retrieved on table either under kebab menu, or on the right of the ta
 
 Some information before using the public API:
 
+- ApiToken must be present in the `Authorization` header after the `bearer ` prefix (exactly `bearer ` with one space)
```

Comment:
- The `Authorization` header is mandatory. Its value must be `bearer ` concatenated with the value of your ApiToken. Beware of the white space after the `bearer` prefix.

Reply:
Note: do we really support both `bearer` and `Bearer`???

Header values are case sensitive and `Bearer` is definitely the standard form, see https://www.rfc-editor.org/rfc/rfc6750#section-6.1.1
That's the form (least unexpected) that we should emphasize.

Reply:
we remove the first 8 characters.

Reply:
> we remove the first 8 characters.

Instead of validating the format? 😱